monthly-contributions/src/main/scala/com/gu/support/workers/lambdas/SendThankYouEmail.scala

```diff
@@ -33,4 +35,14 @@ class SendThankYouEmail(thankYouEmailService: EmailService)
       product = "monthly-contribution"
     )).map(_ => Unit)
   }
+
+  private def putMetric(paymentType: String) =
```

Comment:
As above - merge these two methods and just call toLowerCase() on the paymentType